service/grails-app/migrations/update-mod-license-1-6-1.groovy

```diff
@@ -0,0 +1,29 @@
+databaseChangeLog = {
+  changeSet(author: "John Fereira", id: "2019-05-14-ERM-218-1") {
+    modifyDataType( 
+      tableName: "custom_property_definition", 
+      columnName: "pd_description", 
+      newDataType: "TEXT",
+      confirm: "Successfully updated pd_description column."
+    )
+  }
+
+  changeSet(author: "John Fereira", id: "2019-05-14-ERM-218-2") {
+    modifyDataType( 
+      tableName: "license", 
+      columnName: "lic_description", 
+      newDataType: "TEXT",
+      confirm: "Successfully updated lic_description column." 
+    )
+  }
+
+  changeSet(author: "John Fereira", id: "2019-05-14-ERM-218-3") {
+    modifyDataType( 
```

Comment:
Same again here... 
Although named 'description' this column should be left as is.